src/main/antora/modules/cdi/pages/cdi-basic/cdi-basic002a.adoc

```diff
@@ -0,0 +1,26 @@
+== CDI Lite vs CDI Full
+
+As of CDI version 4.0 in Jakarta EE version 10, the Core CDI functionality has been split into two parts: CDI Lite and CDI Full.
+ 
+CDI Lite provides a subset of the CDI Full functionality with an emphasis on build-time implementations. By leaving out the additional components from CDI Full such as those dealing with runtime reflection, CDI Lite is able to execute in lighter and more restricted environments.
+
+Jakarta EE-compliant application servers will still implement the CDI Full functionality so this change will benefit those developers working in alternate (e.g. cloud-based) environments without affecting those working in a standard JEE environment.
```

Comment:
We want to eliminate usage of acronyms/abbreviations and this includes "JEE". Please write out fully.